SerialAPI.h

```diff
@@ -87,6 +89,25 @@ class SerialAPI: public FB::JSAPIAuto
 			delete [] lpCC;
 		}
 #endif
+#ifdef __unix__
+            DIR *dev = opendir("/dev");
+            struct dirent *ent;
+
+            if (dev != NULL) {
+                while (ent = readdir(dev)) {
+                    std::string name(ent->d_name);
+                    std::string prefix = name.substr(0, 6); 

```

Comment:
@Gregwar what if the length of ```name``` is less then 6? an out_of_range exception will be thrown. Please a a check here. Something like below:
```C++
if (name.length()<=6) {
    continue;
}
```